### ISSUE TITLE
Any way to make the client smaller?

### ISSUE THREAD

**Issue by Betterer** [runelite/runelite#2807]
I know that it has to be 765x503; however, my konduit client takes up about 30% less of my screen than my runelite client and I am wondering if there is any way to make the runelite client the same size as the smaller konduit one.

**Comment by SoyChai** [comment]
Screenshot?

**Comment by permedd** [comment]
yes! i think the client is too big i would like it smaller, i'm on fixed mode as well.
